src/components/app/components/Navbar.js

```diff
@@ -0,0 +1,77 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Avatar,  Dropdown, Layout, Menu } from 'antd';
+
+import AppLink from './AppLink';
+import home from '../../home';
+
+import signin from '../../signin';
+import signup from '../../signup';
+import { LOGO, MARKET_TEXT, NAVBAR, PATHS, USER_AVATAR } from '../constants';
+
```

Comment:
Why did you separate the imports being that they are parts of the app

Reply:
You want to separate the imports of external dependencies from those of internal files. We will go over this during our next meeting.

Reply:
OK